### PR TITLE
added combatlogx and default-gamemode faq

### PR DESCRIPTION
**Pull request by ahoulton**
This pull request adds a CombatLogX tutorial with a how-to set default game mode tutorial.

### CODE REVIEW

**Comment by LukasmanMHdude** [comment]
Hey, you changed a lot of wierd stuff with this, why did you mess with all those other files?

**Comment by ahoulton** [comment]
> 
> 
> Hey, you changed a lot of wierd stuff with this, why did you mess with all those other files?

What files? I only added some stuff, I'm confused.

File: SUMMARY.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 * [Minehut.xyz](README.md)
 * [Contribute](contribute.md)
-* [Sponsors](sponsor.md)
```
Comment (LukasmanMHdude) [inline SUMMARY.md:5]: why did you remove

File: SUMMARY.md [hunk 2]
```diff
@@ -12,9 +11,9 @@
   * [Resource Packs](faq/resourcepacks.md)
   * [Custom Domain](faq/domain.md)
   * [Allowed File types](faq/files.md)
-  * [Reset Server](faq/reset.md)
+  * [Reset Server](faq/reset-server.md)
```
Comment (LukasmanMHdude) [inline SUMMARY.md:14]: Not sure what your doing here

File: SUMMARY.md [hunk 3]
```diff
@@ -12,9 +11,9 @@
...
+  * [Default Gamemode](faq/default-gamemode.md)
   * [Default World](faq/default-world.md)
-  * [Custom Seed](faq/custom-seed.md)
```
Comment (LukasmanMHdude) [inline SUMMARY.md:17]: why did you remove

File: SUMMARY.md [hunk 4]
```diff
@@ -26,7 +25,6 @@
   * [Creating a Void World](faq/void-world.md)
 * [Minehut](popqs3/README.md)
   * [Forums Club](faq/clubs.md)
-  * [Cracked Minecraft](faq/cracked.md)
```
Comment (LukasmanMHdude) [inline SUMMARY.md:29]: why did you remove

**Review by LukasmanMHdude — CHANGES_REQUESTED** [review]
Not sure if your trying to change how our website is laid out, but please make a suggestion to change/remove categories instead of just doing it

**Comment by LukasmanMHdude** [comment]
While I do agree these need to be redone, just deleting them isn't the answer

**Comment by ahoulton** [comment]
> Not sure if your trying to change how our website is laid out, but please make a suggestion to change/remove categories instead of just doing it

Uh, your making no sense... If you mean why some stuff was removed from SUMMARY.md was because I was using a old template. Otherwises, I see no issues.

File: SUMMARY.md (hunk 1 above)
Comment (ahoulton) [reply SUMMARY.md:5]: Old template.

File: SUMMARY.md (hunk 2 above)
Comment (ahoulton) [reply SUMMARY.md:14]: Renamed incase someone wants to make a reset world tutorial.

File: SUMMARY.md (hunk 3 above)
Comment (ahoulton) [reply SUMMARY.md:17]: Have no idea what happened to that.

File: SUMMARY.md (hunk 4 above)
Comment (ahoulton) [reply SUMMARY.md:29]: Uh, I actually don't know where that went.

**Comment by LukasmanMHdude** [comment]
Please just copy and paste everything into a new fork, or fix your current one

**Comment by ahoulton** [comment]
> 
> 
> Please just copy and paste everything into a new fork, or fix your current one

I have resolved on everything you have suggested.

**Comment by ahoulton** [comment]
I'm sorry for this inconvenience, I'm very new on how this works.

**Comment by LukasmanMHdude** [comment]
If you want, I can make the pull req for you

**Comment by ahoulton** [comment]
> 
> 
> If you want, I can make the pull req for you

Only if you want to, but I happy for you to do it.

**Comment by ahoulton** [comment]
Also, closing this due to errors and weird stuff being removed.

**Comment by ahoulton** [comment]
Updated: https://github.com/TeamMH/minehutxyz/pull/16
